sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -674,7 +674,12 @@ def _add_argparse_args(cls, parser):
          'job submission, the files will be staged in the staging area '
          '(--staging_location option) and the workers will install them in '
          'same order they were specified on the command line.'))
-
+    parser.add_argument(
+        '--dataflow_worker_jar',
+        dest='dataflow_worker_jar',
+        type=str,
+        help='Dataflow worker jar.'
+    )
```

Comment:
I'm thinking that the option would be better in `WorkerOptions`, or some other options class related to Dataflow+Portability.

Reply:
Thanks!  Issue addressed but lost the status in this PR due to my sub-optional git operations. 

Opening another PR.